src/Aspire.Hosting/Dashboard/proto/resource_service.proto

```diff
@@ -38,6 +38,10 @@ message ResourceCommand {
     // Clients must return any value provided by the server when invoking
     // the command.
     optional google.protobuf.Value parameter = 4;
+    // A flag that indicates whether the command is highlighted in the UI.
+    bool is_highlighted = 5;
```

Comment:
I think this should be optional, so we don't break the contract. It can safely default to false.

Reply:
In proto, all fields are optional. If it isn't specify then it defaults to the default value (false).

I believe the only impact `optional` keyword has is you can check whether a value was explicitly set. We don't care about that here.